tests/trezor/trezor_tests.cpp

```diff
@@ -958,7 +1018,11 @@ void gen_trezor_base::fix_hf(std::vector<test_event_entry>& events)
     ADD_HARDFORK(m_hard_forks, hf_to_add, hardfork_height);
     add_top_hfork(events, m_hard_forks);
     MDEBUG("Hardfork added at height: " << hardfork_height << ", from " << (int)current_hf << " to " << (int)hf_to_add);
-    rewind_blocks(events, 10, hf_to_add);
+    rewind_blocks(events, hf_to_add == m_top_hard_fork ? 10 : 2, hf_to_add);
```

Comment:
Wallet requires +10 blocks to activate last HF, thus 10 block rewind to get the latest HF active.